bc_obps/service/tests/test_operation_service_v2.py

```diff
@@ -295,3 +310,224 @@ def test_create_or_replace_statutory_declaration():
+            tertiary_naics_code_id=3,
+            activities=[1],
+            process_flow_diagram=MOCK_DATA_URL,
+            boundary_map=MOCK_DATA_URL,
+            equipment_list=MOCK_DATA_URL,
+        )
+        operation = OperationServiceV2.create_or_update_operation_v2(
+            approved_user_operator.user.user_guid, None, payload
+        )
+        # brianna why do i have to refresh? just a test thing or a problem?
+        operation.refresh_from_db()
+        assert Operation.objects.count() == 1
+        assert operation.activities.count() == 1
+        assert operation.documents.count() == 3
+        assert operation.created_by == approved_user_operator.user
+        assert operation.created_at is not None
+        assert operation.updated_at is None
+
+    @staticmethod
+    def test_create_operation_with_multiple_operators():
+        approved_user_operator = baker.make_recipe('utils.approved_user_operator')
+        payload = OperationInformationIn(
+            registration_purpose='Reporting Operation',
+            regulated_products=[1],
+            name="string",
+            type="SFO",
+            naics_code_id=1,
+            secondary_naics_code_id=2,
+            tertiary_naics_code_id=3,
+            activities=[1],
+            process_flow_diagram=MOCK_DATA_URL,
+            boundary_map=MOCK_DATA_URL,
+            equipment_list=MOCK_DATA_URL,
+        )
+        payload.multiple_operators_array = [
+            MultipleOperatorIn(
+                mo_legal_name='i am new',
+                mo_trade_name='new',
+                mo_cra_business_number=111111111,
+                mo_business_structure='General Partnership',
+                mo_bc_corporate_registry_number='ghj1234567',
+                mo_attorney_street_address='Seahorses St',
+                mo_municipality='Ocean Town',
+                mo_province='BC',
+                mo_postal_code='H0H0H0',
+            ),
+            MultipleOperatorIn(
+                mo_legal_name='i am new 2',
+                mo_trade_name='new 2',
+                mo_cra_business_number=111111111,
+                mo_business_structure='General Partnership',
+                mo_attorney_street_address='Lion St',
+                mo_municipality='Zebra Town',
+                mo_province='MB',
+                mo_postal_code='H0H0H0',
+            ),
+        ]
+        operation = OperationServiceV2.create_or_update_operation_v2(
+            approved_user_operator.user.user_guid, None, payload
+        )
+        operation.refresh_from_db()
+        assert Operation.objects.count() == 1
+        assert operation.created_by == approved_user_operator.user
+        assert operation.created_at is not None
+        assert MultipleOperator.objects.count() == 2
+        assert MultipleOperator.objects.all().first().bc_corporate_registry_number == 'ghj1234567'
+        assert MultipleOperator.objects.all().last().bc_corporate_registry_number is None
```

Comment:
I don't think the `all()` is needed - should just be able to do
`MultipleOperator.objects.first()` and `MultipleOperator.objects.last()`

Reply:
:exploding_head: